docs/igor0806/GamesProject/games.html

```diff
@@ -0,0 +1,156 @@
+                                <option value="MMORPG">MMORPG</option>
+                                <option value="Card Game">Card Game</option>
+                            </select>
+                        </div>
+
+                        <div class="platform-select">
+                            <label for="gamePlatform"></label>
+                            <select id="gamePlatform" name="PlatformSelect" class="select-name" data-type-filter="platform">
+                                <option value="Platform" disabled selected>Platform</option>
+                                <option value="PC">PC (Windows)</option>
+                                <option value="Web">Web Browser</option>
+                            </select>
+                        </div>
+                    </div>
+
+                    <div class="filter_radio">
+                        <input type="radio" class="radio" id="New" name="Radio" data-type-filter="new">
+                        <label for="New" class="New_radio">New</label>
+                        <input type="radio" id="Old" name="Radio" data-type-filter="old">
+                        <label for="Old" class="Old_radio">Old</label>
+                    </div>
+
+                    <div class="filter_search">
+                        <input type="text" id="search" name="search" placeholder="Search" class="search-input" data-type-filter="search">
+                        <label for="search"></label>
+                        <button type="button" id="search-button" class="search-button" data-type-filter="search-button">Search</button>
+                    </div>
+                </div>
+            </div>
+        </form>
+
+        <div class="all_cards">
+            <div class="container">
+                <h2 class="title_about_1">Study project "Games"</h2>
+                <div class="card_load" data-type="loading">
+                    <p class="card_loading_text">Loading...</p>
+                    <p class="loading"></p>
+                </div>
+                <div class="card_cards">
+                    <ul class="card_cards" data-type="games-list"></ul>
+                    <template data-type="card-template">
+                        <div class="card_card">
+                            <div class="card_games">
+                                <div class="card_img">
+                                    <img data-card-img alt="img" src="" width="90px" height="90px">
+                                </div>
+                                <div class="card_title">
+                                    <h3 class="name_game" data-card-title></h3>
+                                    <p class="info_game" data-card-description></p>
+                                </div>
+                            </div>
+                            <div class="card_text">
+                                <ul class="info_game">
+                                    <li data-card-genre></li>
```

Comment:
Please correct the layout 
<img width="1433" alt="image" src="https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/4dcfcef7-73b3-40e0-a51a-cac4a01152fe">